README.md

```diff
@@ -38,3 +40,31 @@ interface only leaves you to implement the gRPC call implementation itself.
 
 Given DeviceLister, DevicePluginManager is capable of handling lifetime of the
 device plugin, it's devices and gRPC server.
+
+## Develop
+
+Build all plugins:
+
+```
+make build
+```
+
+Build specific plugins:
+
+```
+make build-vfio
```

Comment:
Not fan of the implicit build-% and test-% rules. Would be nice to have another review in this regard.